run-demo-delegations.py

```diff
@@ -0,0 +1,275 @@
+#!/usr/bin/env python
+
+import os
+import shlex
+import shutil
+import subprocess
+import sys
+import tempfile
+
+
+NO_PROMPT = False
+REQUIRED_BINARIES = ["git", "gittuf", "ssh-keygen"]
+
+
+def check_binaries():
+    for p in REQUIRED_BINARIES:
+        if not shutil.which(p):
+            raise Exception(f"required command {p} not found")
+
+
+def prompt_key(prompt):
+    if NO_PROMPT:
+        print("\n" + prompt)
+        return
+    inp = False
+    while inp != "":
+        try:
+            inp = input(f"\n{prompt} -- press any key to continue")
+        except Exception:
+            pass
+
+
+def display_command(cmd):
+    print(f"[{os.getcwd()}] $ {cmd}")
+
+def run_command(cmd, expected_retcode=0):
+    retcode = subprocess.call(shlex.split(cmd))
+    if retcode != expected_retcode:
+        raise Exception(f"Expected {expected_retcode} from process but it exited with {retcode}.")
+
+def run_demo():
+    current_dir = os.getcwd()
+    keys_dir = "keys"
+
+    tmp_dir = tempfile.TemporaryDirectory()
+    tmp_keys_dir = os.path.join(tmp_dir.name, keys_dir)
+    tmp_repo_dir = os.path.join(tmp_dir.name, "repo")
+
+    shutil.copytree(os.path.join(current_dir, keys_dir), tmp_keys_dir)
+    os.mkdir(tmp_repo_dir)
+    os.chdir(tmp_repo_dir)
+
+    for key in os.listdir(tmp_keys_dir):
+        os.chmod(os.path.join(tmp_keys_dir, key), 0o600)
+
+    authorized_key_path_git = os.path.join(tmp_keys_dir, "authorized")
+    unauthorized_key_path_git = os.path.join(tmp_keys_dir, "unauthorized.pub")
+
+    authorized_key_path_policy = os.path.join(tmp_keys_dir, "authorized.pub")
+
+    prompt_key("Initialize Git repository")
+    cmd = "git init -b main"
+    display_command(cmd)
+    run_command(cmd)
+
+    prompt_key("Set repo config to use demo identity and test key")
+    cmd = f"git config --local gpg.format ssh"
+    display_command(cmd)
+    run_command(cmd)
+    cmd = f"git config --local commit.gpgsign true"
+    display_command(cmd)
+    run_command(cmd)
+    cmd = f"git config --local user.signingkey {authorized_key_path_git}"
```

Comment:
```suggestion
    cmd = f"git config --local user.signingkey {root_key_path_git}"
```

Let's change this to the root key so that the gittuf initialization commits are signed by the root key.